docs/src/components/DocsMDXProvider.js

```diff
@@ -8,23 +8,40 @@ import MDXPlayground from './MDXPlayground'
  */
 const Components = {}
 
-const generateHeadingId = (props) => {
-  if (props && typeof props.children === 'string') {
-    return `heading_${props.children.toLowerCase().replace(' ', '_')}`
```

Comment:
This was only replacing the first instance of a space character.

Reply:
nice catch